src/sqlfluff/core/default_config.cfg

```diff
@@ -174,7 +174,7 @@ group_by_and_order_by_style = consistent
 unquoted_identifiers_policy = all
 quoted_identifiers_policy = all
 allow_space_in_identifier = False
-additional_allowed_characters = ""
+additional_allowed_characters = None
```

Comment:
Empty quotes don't return an empty string. They literally return the string `""`, which I believe is not what we want -- i.e., we don't want to allow `"` in names.